packages/@okta/vuepress-site/docs/concepts/authentication/index.md

```diff
@@ -7,80 +7,43 @@ meta:
 
 To find out more about Okta's OAuth 2.0 and OpenID Connect implementations see [OAuth 2.0 and OpenID Connect](/docs/concepts/auth-overview/).
 
 ### Sign-In Widget
 
-The [Okta Sign-in Widget](/code/javascript/okta_sign-in_widget/)
-provides an embeddable Javascript sign-in implementation that can
-easily be embedded into your customized login page.  The Sign-in
-widget carries the same feature set in the standard Okta sign-in page
-of every tenant - with the added flexibility to change the
-look-and-feel.  Included in the widget is support for password reset,
-forgotten password and strong authentication - all of which are  driven
-by policies configured in Okta.  Developers don't have to write a
-single line of code to trigger these functions from within the widget.
-For consumer facing sites, social providers are also supported in the
-widget.
+The [Okta Sign-in Widget](/code/javascript/okta_sign-in_widget/) provides an embeddable Javascript sign-in implementation that can easily be embedded into your customized login page.  The Sign-in widget carries the same feature set in the standard Okta sign-in page of every tenant along with the added flexibility to change the look and feel.  Included in the widget is support for password reset, forgotten password and strong authentication functionality - all of which are  driven by policies configured in Okta.  Developers don't have to write a single line of code to trigger these functions from within the widget. For consumer facing sites, social providers are also supported in the widget.
 
 ### Auth SDK
 
-For those who are building a Javascript front end or Single Page App
-(SPA), the light-weight, JavaScript-based [Okta Auth SDK](/code/javascript/okta_auth_sdk/)
-gives you the added control beyond our sign-in widget to cater to your
-needs.  This Javascript SDK provides all the standard login support
-including password management and strong authentication.  In addition,
-social providers and OpenID Connect are also supported through the SDK where
-the appropriate ID tokens are returned for downstream authentication
-and authorization needs.
+For developers building a Javascript front end or Single Page App (SPA), the light-weight, JavaScript-based [Okta Auth SDK](/code/javascript/okta_auth_sdk/) gives you additional control beyond our sign-in widget.  This Javascript SDK provides all the standard login support, including password management and strong authentication.  In addition, social providers and OpenID Connect are also supported through the SDK, where the appropriate ID tokens are returned for downstream authentication and authorization needs.
```

Comment:
Fixing some things:

For developers building a Javascript front end or Single Page App (SPA), the light-weight, JavaScript-based [Okta Auth SDK](/code/javascript/okta_auth_sdk/) gives you additional control beyond our Sign-In Widget.  This Javascript SDK provides all the standard sign-in support, including password management and strong authentication. Additionally, social providers and OpenID Connect are supported through the SDK, where the appropriate ID tokens are returned for downstream authentication and authorization needs.

Reply:
Changes made.